packages/ffe-cards-react/src/IconCard/IconCard.tsx

```diff
@@ -12,6 +14,8 @@ export type IconCardProps<As extends ElementType = 'div'> = Omit<
     icon: ReactElement;
     /** Smaller icon and less space */
     condensed?: boolean;
+    /** Position icon at left (default) or right of the card content */
+    iconPosition?: IconPosition;
```

Comment:
Kan sikkert bare vare sånn :) Jag burde skrevet sånn i min kommentar.

```suggestion
    iconPosition?: 'right' | 'left';
```